pkglistgen.py

```diff
@@ -70,7 +70,7 @@
 SCRIPT_PATH = os.path.dirname(os.path.realpath(__file__))
 ARCHITECTURES = ['x86_64', 'ppc64le', 's390x', 'aarch64']
 DEFAULT_REPOS = ("openSUSE:Factory/standard")
-PRODUCT_SERVICE = '/usr/lib/obs/service/create_single_product'
+PRODUCT_SERVICE = '/home/leaper/obs-service-product_converter/create_single_product'
```

Comment:
This would seem like a problem.

Reply:
Just as your code is a problem for me - fair treat, no? :)

Reply:
A joke I presume.